src/config.rs

```diff
@@ -97,7 +101,20 @@ where
         }
     }
 
-    let triggers = HashMap::<Trigger, Vec<Modification>>::deserialize(deserializer)?;
+    let mut triggers = HashMap::<Trigger, Vec<Modification>>::deserialize(deserializer)?;
+    // If there are any `approve` triggers, add `unapprove` triggers as well.
```

Comment:
This logic is taken from `homu`?

Reply:
I couldn't find any logical way to remove the labels added after approving a PR, so I just add a section labels to add/remove after unapproving a PR as well.